src/common/options/rgw.yaml.in

```diff
@@ -3989,3 +3989,11 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_op_counters_dump_expiration
+  type: uint
+  level: advanced
+  desc: Time in seconds greater than 0 after which performance counters that are not updated will not be dumped
+  default: 0
+  services:
+  - rgw
+  with_legacy: true
```

Comment:
if this expiration is needed, then it should be on by default

might help to mention the relationship between this option and exporter_stats_period
```yaml
  see_also:
  - exporter_stats_period
```

Reply:
We're going to figure out what a good default would be for the dashboards later this week and might adjust this accordingly